CarRentals/CarRentalsWebAPI/Models/CarRentalsContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CarRentals.Models;
+using Microsoft.EntityFrameworkCore;
+using Models;
+
+namespace CarRentalsWebAPI.Models
+{
+    public class CarRentalsContext : DbContext
+    {
+        public CarRentalsContext(DbContextOptions<CarRentalsContext> options) : base(options)
+        {
+
+        }
```

Comment:
we can consolidate all of this in a single line if you want, if you don't like that then just remove the empty lines. this applies to all constructors

Reply:
Perfect, I modified to have it in only one line.